src/components/Text/src/Text.vue

```diff
@@ -227,10 +227,14 @@ export default {
 		 * Detect if the text is center aligned and add left padding
 		 * to balance out the letter spacing
 		 */
-		detectAlignCenter() {
+		detectAlignCenterAndLetterSpacing() {
+			if (!this.letterSpacing && !this.resolvedLetterSpacing) {
```

Comment:
i don't think you need the `!this.letterSpacing` here, I think just `!this.resolvedLetterSpacing` is enough (since it checks `this.letterSpacing` under the hood).